examples/custom-metrics/client.py

```diff
@@ -0,0 +1,60 @@
+import os
+
+import flwr as fl
+import numpy as np
+import tensorflow as tf
+from sklearn.metrics import accuracy_score, recall_score, precision_score, f1_score
+
+
+# Make TensorFlow log less verbose
+os.environ["TF_CPP_MIN_LOG_LEVEL"] = "3"
+
+
+# Load model and data (MobileNetV2, CIFAR-10)
+model = tf.keras.applications.MobileNetV2((32, 32, 3), classes=10, weights=None)
+model.compile("adam", "sparse_categorical_crossentropy", metrics=["accuracy"])
+(x_train, y_train), (x_test, y_test) = tf.keras.datasets.cifar10.load_data()
```

Comment:
Should we use Flower Datasets here?

Reply:
https://github.com/adap/flower/pull/1958/commits/41043b74fe6921f4633cdc5722802ee336c9a5b8#diff-ecd736289c63680fe85bba7c7a5821ecf2a4fbd519f50c9f395c343372d9e87dR17-R27